models/prices/ethereum/prices_ethereum_tokens.sql

```diff
@@ -197,6 +197,7 @@ FROM
     ("iotx-iotex", "ethereum", "IOTX", "0x6fb3e0a217407efff7ca062d46c26e5d60a14d69", 18),
     ("isla-insula", "ethereum", "ISLA", "0x697ef32b4a3f5a4c39de1cb7563f24ca7bfc5947", 18),
     ("itc-iot-chain", "ethereum", "ITC", "0x5e6b6d9abad9093fdc861ea1600eba1b355cd940", 18),
+    ("itgr-integral", "ethereum", "ITGR", "0x782534550e2553a42cdff8d5a94066d8c7b6729b", 18),
```

Comment:
it appears this token is not available on coinpaprika:
https://api.coinpaprika.com/v1/coins/itgr-integral

two options:
- remove from this PR, let new dex tables build without pricing info on this token (the opposite token in a trade could still compute usd amount, based on coalesce used)
- look to add on coinpaprika (https://coinpaprika.com/add/), wait for it to be added before merging this entire PR

Reply:
Hi @IntegralHQ, do you think the correct token-id is `itgr-integral1132`?
https://api.coinpaprika.com/v1/coins/itgr-integral1132
But the token address listed in the API is unmatched to the address in the file.
https://etherscan.io/token/0xd502f487e1841fdc805130e13eae80c61186bc98